modular_skyrat/master_files/code/datums/id_trim/jobs.dm

```diff
@@ -68,13 +68,10 @@
 	subdepartment_color = COLOR_GREEN
 	sechud_icon_state = SECHUD_NT_CONSULTANT
 	extra_access = list()
-	minimal_access = list(ACCESS_SECURITY, ACCESS_BRIG_ENTRANCE, ACCESS_COURT, ACCESS_WEAPONS,
-				ACCESS_MEDICAL, ACCESS_PSYCHOLOGY, ACCESS_ENGINEERING, ACCESS_CHANGE_IDS, ACCESS_AI_UPLOAD, ACCESS_EVA, ACCESS_COMMAND,
-				ACCESS_ALL_PERSONAL_LOCKERS, ACCESS_MAINT_TUNNELS, ACCESS_BAR, ACCESS_JANITOR, ACCESS_CONSTRUCTION, ACCESS_MORGUE,
-				ACCESS_CREMATORIUM, ACCESS_KITCHEN, ACCESS_HYDROPONICS, ACCESS_LAWYER,
-				ACCESS_MECH_ENGINE, ACCESS_MECH_SCIENCE, ACCESS_MECH_SECURITY, ACCESS_MECH_MEDICAL,
-				ACCESS_THEATRE, ACCESS_CHAPEL_OFFICE, ACCESS_LIBRARY, ACCESS_RESEARCH, ACCESS_VAULT, ACCESS_MINING_STATION,
-				ACCESS_HOP, ACCESS_RC_ANNOUNCE, ACCESS_KEYCARD_AUTH, ACCESS_GATEWAY, ACCESS_MINERAL_STOREROOM, ACCESS_AUX_BASE, ACCESS_TELEPORTER, ACCESS_CENT_GENERAL)
```

Comment:
While I'm all for removing access to, say, **Change IDs** or **AI Upload**, a large amount of these are intended to give the NTC an "HR" feel, where they can view a workplace to ensure it's not engaging in policy breaking activity. They should keep access to the primary rooms in departments, i.e. Medical/Research/Engineering/Security/Cargo.
- A prisoner should be able to trust that the NTC can come check on them to ensure they're being treated legally
- A patient should see that Medbay is keeping them in good practice and not breaking policies
- The crew should know that an engineering department who's behaving unsafely can be inspected by somebody with a direct line
- Complaints about missing cargo orders should be open for investigation by the person tying NT to the station, as the FTU is tied by cargo
- The main facet of an NT station, the R&D, should be viewable by an official who can address any needs or concerns they have

Reply:
Ultimately, the NTC is an advisor with no actual authority anymore under the new policy; they are a role about working with people to do what you want and if that fails, faxing Central and hoping they agree with you enough to send someone to actually enforce your opinion since you cannot do it yourself. Combine this with the recent policy shift to keep heads out of eachother's departments, and NT reps really shouldn't be wandering around poking into departments that, by the lore and by the policy, especially given those departments may be completely unrelated to NT in any way, shape, or form.